scripts/paper/generate_stwo_publication_bundle.sh

```
@@ -0,0 +1,211 @@
+    "",
+    "These artifacts remain intentionally narrow. They do **not** prove full standard-softmax transformer inference, recursive aggregation, or production-scale decoding. Their role is to provide a frozen second evidence tier for the paper's experimental `stwo` path.",
+    "Timing rows in the accompanying index are local wall-clock bundle runs under an existing cargo build cache and should be read as artifact metadata rather than a normalized backend benchmark.",
+    "",
+    "See `APPENDIX_ARTIFACT_INDEX.md` for exact hashes, timings, and metadata.",
+]
+readme_md.write_text("\n".join(readme_lines) + "\n")
+PY
+
+(
+  cd "$BUNDLE_DIR"
+  shasum -a 256 *.json benchmarks.tsv manifest.txt commands.log APPENDIX_ARTIFACT_INDEX.md README.md > "$SHA256S"
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)